### PR TITLE
Release first public version of Snap Adapters

### PR DESCRIPTION
**Pull request by rsatrasala**

### CODE REVIEW

File: Snap/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+## Changelog
+* 1.0.6.0
```
Comment (rsatrasala) [inline Snap/CHANGELOG.md:2]: To do
Cleanup changelog to only first release

File: Snap/SnapAdAdapterConfiguration.m [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+}
+
+- (void)initializeNetworkWithConfiguration:(NSDictionary<NSString *, id> *)configuration
+                                  complete:(void(^)(NSError *))complete {
+    
+    NSString * appId = configuration[kSAKAppId];
+    
+    if ([appId length] == 0) {
+        NSError * error = [NSError errorWithDomain:kAdapterErrorDomain code:SAKAdpaterErrorCodeMissingAppId userInfo:@{ NSLocalizedDescriptionKey: @"Snap Ad Kit initialization skipped. The appId is empty. Ensure it is properly configured on the MoPub dashboard." }];
+        MPLogEvent([MPLogEvent error:error message:nil]);
```
Comment (hmertkahraman) [inline Snap/SnapAdAdapterConfiguration.m:56]: We can also call `complete(error)` here

File: Snap/SnapAdBannerCustomEvent.m [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+                     self.slotId);
+    } else {
+        typeof(self) __weak weakSelf = self;
+        
+        [SnapAdAdapterConfiguration initSnapAdKit:info
+                                         complete:^(NSError *error) {
+            if (!error) {
+                [weakSelf.sakAdView loadRequest:[configurationBuilder build]];
+                
+                MPLogAdEvent([MPLogEvent adLoadAttemptForAdapter:NSStringFromClass(self.class) dspCreativeId:nil dspName:nil],
```
Comment (hmertkahraman) [inline Snap/SnapAdBannerCustomEvent.m:73]: We can call log statement before the load request.

**Comment by p158276** [comment]
This PR has been merged according to the commit https://github.com/mopub/mopub-ios-mediation/commit/9c7864fefa3610ed4e2fe402e7a5590ab62958de

**Pull request closed without merging**